docs/source/payout.rst

```diff
@@ -0,0 +1,51 @@
+Distribute Dividends
+====================
+
+The command line tool supports a simple CSV input to make payouts to shareholders.
+
+* Shares are exchanged here for ether or tokens like Gemini USD.
+
+* ``payout-distribute`` takes in the input csv with investor info.
+
+* ``create-holders-payout-csv`` is an optional command that can be used to generate the CSV. This CSV contains investor with
+    their respective shares. This basically fetches data by running ``token-scan`` in the background.
```

Comment:
Do we need a wrapper? Could we improve `token-scan` by including CSV support?

Reply:
These are two commands have different purpose. I don't want to combine the two into one.